tests/integration/test_config.py

```diff
@@ -1,8 +1,10 @@
 import pytest
-import src
 
+import src
+from src.utils.envutils import get_config_path, config_dir_key
 
 @pytest.mark.order(2)
 @pytest.mark.integration
 def test_config() -> None:
-    src.types.Config.load()
+    config_path = get_config_path(config_dir_key())
```

Comment:
Don't do this chained call. Simply do `get_config_path()`. The argument is not necessary when it is always the same value (should only be changed via env var).

Reply:
updated